ansible/roles/owncloud/tasks/ldap.yml

```diff
@@ -14,6 +14,8 @@
   register: owncloud__register_ldap_default_config_exits
   become: True
   become_user: '{{ owncloud__app_user }}'
+  failed_when: (owncloud__register_ldap_default_config_exits.rc != 0 and
```

Comment:
Sorry for the ~3 time back and forth. `owncloud__register_ldap_default_config_exits` should probably mean `owncloud__register_ldap_default_config_exists`.